STANDARDS.md

```diff
@@ -29,6 +29,19 @@ Make sure your plugin follows the SoC design principle and has different functio
 
 Make effective use of namespacing since it will allow co-existence of ambiguous names / classes with the same name and help avoid collisions.
 
+### Recommended Code Style
```

Comment:
Logically I would prefer this to be under #Plugin Standards instead of Development Practices.

Reply:
I can do that but shouldn’t formatting be under Development Practices?